Lesson21/todo-app/sahra/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>To-Do App</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <main class="container">
+      <form class="todo-form" id="todoForm">
+        <h1>To-Do List App</h1>
+        <label for="todoInput">What needs to be done?</label>
+        <input placeholder="Do homework" id="todoInput" class="todo-form-item"/>
```

Comment:
1. Increase the size of the input field, it now looks disproportional comparing to other elements on the page.
2. Add padding to the input field
3. Add margin between input field and the submit button
4. Update color of the input field to match the rest of the page.